src/trace/listener.ts

```diff
@@ -139,8 +140,14 @@ export class TraceListener {
       }
       if (this.inferredSpan) {
         this.inferredSpan.setTag("http.status_code", statusCode);
+
+        if (statusCode?.length === 3 && statusCode?.startsWith("5")) {
+          this.wrappedCurrentSpan.setTag("error", "5xx Server Error");
```

Comment:
is it normal that this tag does not show up in the snapshots?

Reply:
Good question. I thought about this too. I think `error` tag is treated differently. Asked in slack, if no one knows other reasons, I think special treatment is probably the reason.

Reply:
Updated the value of this tag to `1`. The theory is that this `error` tag is transformed to ` "error": 1` instead of `meta` field like other normal tags.
Tested the change in sandbox and works as expected (and the same as before).
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/47579703/177424707-330d9b28-b3cb-4907-93a3-b154a9f4379c.png">